### DISCUSSION TITLE
Boundary Box request parameter for Transitland v2 REST API

### DISCUSSION THREAD

**Discussion by uq-bfitzpatrick** [transitland/transitland#359] [Q&A] [answered]
Hello,

The Transitland v1 Datastore API included a query parameter `bbox` allowing queries within a set of bounding box coordinates e.g. for `/api/v1/stops`.

Does the Transitland v2 REST API lack a `bbox` request parameter e.g. for `/api/v2/rest/stops` ? 
(I can't see it in the documentation).

If so, would your team be amenable to a feature request or was this a design decision?

Thank you for your time.

**Comment by drewda** [comment, marked as answer]
Hi,

Yes, we initially decided to limit geometric searches in the v2 REST API to radius-only, in order to place limits of the search extent and ensure queries are sufficiently fast.

We're glad to take this feature request into account and consider adding bounding box search to v2 REST API endpoints in the future (with some limits set on the maximum bbox size and further profiling to make sure the queries remain efficient). If/when we add bbox searches, we'll update this thread. Thanks.

**Comment by uq-bfitzpatrick** [comment]
Thank you for this information, it is good to know and we will plan accordingly.
We will also remain interested in bounding box based search functionality being added to the API.
